### PR TITLE
Rename code shortcode to code_sample

### PR DESCRIPTION
**Pull request by mengjiao-liu**
Rename code shortcode to code_sample

see https://github.com/kubernetes/website/issues/42203#issuecomment-1691439368
> It's not too late to switch now, I looked at it and now only zh-cn downstream uses code shortcode.
After the Chinese synchronization is completed again, I can remove the `code` shortcode.

[preview page](https://deploy-preview-42703--kubernetes-io-main-staging.netlify.app/docs/contribute/style/hugo-shortcodes/#source-code-files).

### CODE REVIEW

File: content/en/docs/contribute/style/hugo-shortcodes/index.md [hunk 1]
```diff
@@ -273,33 +273,33 @@ Renders to:
...
 
 ```none
-{{%/* code file="configmap/configmaps.yaml" */%}}
+{{%/* code_sample file="configmap/configmaps.yaml" */%}}
 ```
 
-The legacy `{{%/* codenew */%}}` shortcode is being replaced by `{{%/* code */%}}`.
-Use `{{%/* code */%}}` in new documentation.
+The legacy `{{%/* codenew */%}}` shortcode is being replaced by `{{%/* code_sample */%}}`.
+Use `{{%/* code_sample */%}}` in new documentation.
```
Comment (sftim) [inline content/en/docs/contribute/style/hugo-shortcodes/index.md:302]:
```suggestion
The legacy `{{%/* codenew */%}}` shortcode is being replaced by `{{%/* code_sample */%}}`.
Use `{{%/* code_sample */%}}` (not `{{%/* code */%}}`) in new documentation.
```

**Review by sftim — COMMENTED** [review]
Thanks!

LGTM. We'll need to agree on what approach to take.

File: content/en/docs/contribute/style/hugo-shortcodes/index.md (hunk 1 above)
Comment (mengjiao-liu) [reply content/en/docs/contribute/style/hugo-shortcodes/index.md]: It is indeed better to clarify. Updated. Thanks.

**Comment by Vikash-8090-Yadav** [comment]
LGTM
Thanks.

**Comment by mengjiao-liu** [comment]
Changes based on the latest main branch.

@tengqm @sftim  for approval

File: content/en/docs/contribute/style/hugo-shortcodes/index.md [hunk 2]
```diff
@@ -273,33 +273,33 @@ Renders to:
...
 
 ```none
-{{%/* code file="configmap/configmaps.yaml" */%}}
+{{%/* code_sample file="configmap/configmaps.yaml" */%}}
 ```
 
-The legacy `{{%/* codenew */%}}` shortcode is being replaced by `{{%/* code */%}}`.
-Use `{{%/* code */%}}` in new documentation.
+The legacy `{{%/* codenew */%}}` shortcode is being replaced by `{{%/* code_sample */%}}`.
+Use `{{%/* code_sample */%}}` (not `{{%/* code */%}}`) in new documentation.
```
Comment (sftim) [inline content/en/docs/contribute/style/hugo-shortcodes/index.md:302]:
```suggestion
Use `{{%/* code_sample */%}}` (not `{{%/* codenew */%}}` or `{{%/* code */%}}`) in new documentation.
```

**Review by sftim — COMMENTED** [review]
/lgtm

File: content/en/docs/contribute/style/hugo-shortcodes/index.md (hunk 2 above)
Comment (mengjiao-liu) [reply content/en/docs/contribute/style/hugo-shortcodes/index.md]: Updated.

**Comment by sftim** [comment]
Thanks @mengjiao-liu

**Pull request merged**